aten/src/ATen/native/native_functions.yaml

```diff
@@ -3675,20 +3675,20 @@
     MkldnnCPU: mkldnn_max_pool3d_backward
   autogen: mkldnn_max_pool3d_backward.out
 
-- func: quantized_max_pool1d(Tensor self, int[1] kernel_size, int[1] stride=[], int[1] padding=0, int[1] dilation=1, bool ceil_mode=False) -> Tensor
+- func: max_pool1d(Tensor self, int[1] kernel_size, int[1] stride=[], int[1] padding=0, int[1] dilation=1, bool ceil_mode=False) -> Tensor
```

Comment:
`max_pool1d` already exists. Why do you want to rename it?

Reply:
Hi @jgong5 ,
I will make use of the existing max_pool1d kernel. Thanks for your comment